src/store/swapOrderSlice.ts

```diff
@@ -94,6 +103,7 @@ export const swapOrderStatusListener = (listenerMiddleware: typeof listenerMiddl
                   : 'Waiting for confirmation from signers of your Safe',
               groupKey,
               variant: 'info',
+              link: link ? link : undefined,
```

Comment:
I think if we initialize the `link` as 
`let link = undefined`
we could get rid of all these checks and simply do in all these lines

```suggestion
              link,
```